int32 workmode
int32 pawmode
int32 turnmode
float32 speed
float32 angle
